packages/daimo-common/src/model.ts

```diff
@@ -128,6 +128,7 @@ export interface ProposedSwap {
   fromAcc: EAccount;
   receivedAt: number;
   toAmount: number; // in native USDC units
+  expiresAt: number;
```

Comment:
maybe cacheUntil to distinguish from deadline

Reply:
done